examples/trading/api/README.md

```diff
@@ -0,0 +1,130 @@
+# Escrow API Demo
+
+This demo is built to showcase how we can build an event indexer + API
+to easily & efficiently serve on-chain data for our app.
+
+The demo indexer uses polling to watch for new events.
+
+Everything is pre-configured on testnet, but can be tweaked to work on any other network.
+You can easily change the network by creating a `.env` file with the variable `NETWORK=<mainnet|testnet|devnet|localnet>`
```

Comment:
```suggestion
You can change the network by creating a `.env` file with the variable `NETWORK=<mainnet|testnet|devnet|localnet>`
```
Just FYI, I always suggest never using the word 'easy' in tech docs because if someone has trouble for whatever reason, then you're essentially insulting them.